### PR TITLE
Upgrade/dependencies

### PR DESCRIPTION
**Pull request by lifuzu**
1. Upgrade the defunct dependendencies;
2. Add release and changelog;
3. Tune the AndroidManifest.xml to the right place according to the new plugin requirements;
4. Pump up the version to 1.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+//Continue from top
+
+## Release version 1.0.0 (September, 6th, 2017)
+
+* Based on 0.6.2-b0-SNAPSHOT, this release upgraded some of defunct dependencies
```
Comment (stephanenicolas) [inline CHANGELOG.md]: what is defunct ? obsolete ? (having a common vocabularies is better)

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+1. check `mvn clean install` works well
+2. type `mvn clean release:prepare`
+3. type `mvn clean release:perform`
+
+If step 2 to X fails, type `mvn release:rollback` followed by `mvn release:clean`. A tag might have been pushed and will need to be erased manually on the remote repo, or this follow the instruction at https://nathanhoad.net/how-to-delete-a-remote-git-tag.
+
+### for multi-module releases, for example 1.0.0
+```
+mvn --batch-mode -Dtag=robo-remote-1.0.0 release:prepare \
```
Comment (stephanenicolas) [inline RELEASING.md]: this looks strange to me. You shouldn't need this.

Comment (stephanenicolas) [reply RELEASING.md]: I prefer to have an interactive and simpler command.

File: RoboRemoteServer/apklib/pom.xml [hunk 3]
```diff
@@ -61,6 +61,17 @@
         <version>0.6.2-b0-SNAPSHOT</version>
```
Comment (stephanenicolas) [inline RoboRemoteServer/apklib/pom.xml]: this should be stable too

File: RoboRemoteServer/pom.xml [hunk 4]
```diff
@@ -61,7 +61,17 @@
...
+            <url>http://repo.boundlessgeo.com/main/</url>
+        </repository>
+        <repository>
+            <id>google</id>
+            <url>https://maven.google.com</url>
+        </repository>
+    </repositories>
+
     <properties>
         <com.groupon.roboremote-version>0.6.2-b0-SNAPSHOT</com.groupon.roboremote-version>
```
Comment (stephanenicolas) [inline RoboRemoteServer/pom.xml]: shouldn't this be stable, and be used in the poms ?

File: RoboRemoteServerCommon/pom.xml [hunk 5]
```diff
@@ -61,6 +61,13 @@
...
 
+    <repositories>
+        <repository>
+            <id>boundlessgeo</id>
+            <url>http://repo.boundlessgeo.com/main/</url>
+        </repository>
+    </repositories>
+
     <properties>
         <com.groupon.roboremote-version>0.6.2-b0-SNAPSHOT</com.groupon.roboremote-version>
```
Comment (stephanenicolas) [inline RoboRemoteServerCommon/pom.xml]: same

File: UIAutomatorServer/pom.xml [hunk 6]
```diff
@@ -29,6 +29,17 @@
...
+            <url>http://repo.boundlessgeo.com/main/</url>
+        </repository>
+        <repository>
+            <id>google</id>
+            <url>https://maven.google.com</url>
+        </repository>
+    </repositories>
+
     <properties>
         <com.groupon.roboremote-version>0.6.2-b0-SNAPSHOT</com.groupon.roboremote-version>
```
Comment (stephanenicolas) [inline UIAutomatorServer/pom.xml]: same

File: pom.xml [hunk 7]
```diff
@@ -36,7 +36,7 @@
 
     <groupId>com.groupon.roboremote</groupId>
     <artifactId>roboremote</artifactId>
-    <version>0.6.2-b0-SNAPSHOT</version>
+    <version>1.0.1-SNAPSHOT</version>
```
Comment (stephanenicolas) [inline pom.xml]: I am not sure to understand this PR. So you released version 1.0.0 already and this is the PR to restore GH to a snapshot ?

File: CHANGELOG.md (hunk 1 above)
Comment (lifuzu) [reply CHANGELOG.md]: 'obsolete' should be better.

File: RoboRemoteServer/pom.xml (hunk 4 above)
Comment (stephanenicolas) [reply RoboRemoteServer/pom.xml]: Remove this property.

Comment (stephanenicolas) [reply RoboRemoteServer/pom.xml]: use `<version>${project.version}</version>` on line 110 & 115.

**Pull request merged**